src/DataTypes.php

```diff
@@ -68,7 +68,7 @@ public static function encodeBinary($param): array
                 $type = self::MYSQL_TYPE_DOUBLE;
                 break;
             case "string":
-                $type = self::MYSQL_TYPE_LONG_BLOB;
+                $type = self::MYSQL_TYPE_STRING;
```

Comment:
Thanks! We've merged #113 instead; is this still something we need to change?

Reply:
Hello. It looks better. Can't confirm any issues for now. It can be closed for now. Thanks!

Reply:
Thanks for confirming!